docker-compose.shell.yml

```diff
@@ -0,0 +1,8 @@
+# # Mount code for hot reloading and cache for faster compile in dev mode.
+version: '3.1'
+
+services:
+  civiform:
+    command: ""
```

Comment:
Empty string feels like a "magic value" can you comment on what behavior it's causing?  Is this meant to unset the base definition?

Reply:
since this file is used to override other docker-compose config files my guess this is clearing any value that was set in one of the others?